articles/migrations/guides/linkedin-api-deprecation.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Migration to LinkedIn API V2
+description: This article covers the LinkedIn API deprecation and how to update your Auth0 LinkedIn Connection.
+toc: true
+contentType:
+  - how-to
+useCase:
+  - add-login
+  - migrate
+---
+
+# Migration to LinkedIn API V2
+
+In December 2018, LinkedIn [deprecated the current version of their sign-in API](https://engineering.linkedin.com/blog/2018/12/developer-program-updates). The final shutdown date is set for March 1st, 2019. LinkedIn is replacing the sign-in API with [a new version that has some key differences](https://docs.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/migration-faq?context=linkedin/consumer/context).
```

Comment:
Instead of using the terms current version and new version, maybe use versions 1.0 and 2.0?